src/component/Landing/Main.js

```
@@ -39,100 +52,120 @@ const Main = () => {
+          <div className="carousel-indicators">
+            {getTopBannerList?.data?.length > 0
+              ? getTopBannerList?.data?.map((item, i) => {
+                  return (
+                    <button
+                      type="button"
+                      data-bs-target="#carouselExampleIndicators"
+                      data-bs-slide-to={i}
+                      className={`${i == 0 ? "active" : ""}`}
+                      aria-current={`${i == 0 ? "true" : "false"}`}
+                      aria-label={`Slide ${i + 1}`}
+                    />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)